src/devtools/client/webconsole/components/Input/JSTerm.js

```diff
@@ -132,9 +132,10 @@ class JSTerm extends React.Component {
     }
   };
 
-  onEnter = () => {
+  onEnter = async () => {
     if (!this.showAutocomplete()) {
-      this.execute();
+      // this.execute();
+      this.props.eagerEvalExpression(this.state.value);
```

Comment:
we're actually okay doing a normal evaluation on enter, the key is to use it on key press and then use it for autocomplet

Reply:
oh, this was just for my testing. i haven't lost it yet :P